lib/msal-browser/docs/working-with-b2c.md

```diff
@@ -0,0 +1,104 @@
+# Working with MSAL.js and Azure AD B2C
+
+> :warning: Before you start here, make sure you understand [how to initialize an app object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md) and [working with resources and scopes](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/resources-and-scopes.md). We also recommend general familiarity with **Azure AD B2C**. See the [B2C documentation](https://docs.microsoft.com/azure/active-directory-b2c/technical-overview) for more.
+
+**MSAL.js supports** authentication with **social** (Microsoft, Google, Facebook etc.) and **local** (Azure AD work and school) identities using **Azure AD B2C** (*B2C* for short). When developing B2C apps with **MSAL.js**, there are a few important details to keep in mind.
```

Comment:
@derisen In B2C parlance, a **local** account is a consumer account created using email/password (or username/password).

https://docs.microsoft.com/azure/active-directory-b2c/technical-overview#consumer-accounts

From the doc:

### Consumer accounts

With a *consumer* account, users can sign in to the applications that you've secured with Azure AD B2C. Users with consumer accounts can't, however, access Azure resources, for example the Azure portal.

A consumer account can be associated with these identity types:

* **Local** identity, with the username and password stored locally in the Azure AD B2C directory. We often refer to these identities as "local accounts."
* **Social** or **enterprise** identities, where the identity of the user is managed by a federated identity provider like Facebook, Microsoft, ADFS, or Salesforce.

Reply:
Ah, my bad -fixed!